data/core/units/dunefolk/Blademaster.cfg

```diff
@@ -17,59 +14,47 @@
     {AMLA_DEFAULT}
     cost=52
     usage=fighter
-    description= _ "Distinguished for their bravery and skill, Blademasters are the leaders among the Dunefolk's ranks. Their exploits are well known among their cohorts, inspiring soldiers to fight for their cause. Despite this, they are often found leading from the front, taking down their foes with sword and shield."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MARKSMAN}+{SPECIAL_NOTES_SLOW}
-    die_sound={SOUND_LIST:ELF_HIT}
-    {DEFENSE_ANIM "units/dunefolk/blademaster.png" "units/dunefolk/blademaster.png" {SOUND_LIST:ELF_HIT} }
-    {TRAIT_FEARLESS}
+           description= _ "Alternate Version by Tom_Of_Wesnoth
+
+           For the most renown of Dunefolk swordsmen, the honorific title Blademaster is the highest of all accolades. To earn such a title is to bring great honour upon oneself and one’s line, and to be recognised among the Dunefolk as a fearsome warrior. A Blademaster in combat is a sight to see. In an open field, the Blademaster strikes with a speed and ferocity that is unmatched among the Dunefolk soldiery; while fighting in constrained spaces, however, the Blademaster can land devastating blows with alarming accuracy.
+           
+           DEFAULT
```

Comment:
Again something strange is in description string. Also consider that whitespace is preserved.

Reply:
This one needs finalizing, apparently.